src/frontend/src/components/RecordSearch/index.tsx

```diff
@@ -2,14 +2,104 @@ import React from 'react';
 import { connect } from "react-redux";
 import { AppState } from "../../redux/store";
 import { loadRecords } from "../../redux/records/actions";
+import moment from 'moment';
+
 
 // This is a placeholder component. I created it to prototype
 // out the general structure we'll need when working with
 // Records using Redux.
 
 class RecordSearch extends React.Component {
+  state = {
+    firstName: null,
+    lastName: null,
+    dateOfBirth: '',
+    error: null,
+    validForm: false
+  }
+
+  handleChange = (e: React.BaseSyntheticEvent ) => {
+    console.log(e)
```

Comment:
Oops! Is there a linter set up for this project. If so, could you tell me how to run it? I'd like to avoid these kinds of errors in the future. Thanks!

Reply:
We don't have a linter set up yet! PRs to add a linter are welcome :) 

I just added an Issue for this: https://github.com/codeforpdx/recordexpungPDX/issues/141